q2_demux/_summarize/_visualizer.py

```diff
@@ -55,9 +55,18 @@ def _link_sample_n_to_file(files, counts, subsample_ns):
 
 def _subsample_paired(fastq_map):
     qual_sample = collections.defaultdict(list)
+    seq_len = None
     for fwd, rev, index in fastq_map:
         file_pair = zip(_read_fastq_seqs(fwd), _read_fastq_seqs(rev))
         for i, (fseq, rseq) in enumerate(file_pair):
+            if not seq_len:
```

Comment:
Change to `if seq_len is None:`

Technically if the first sequence has no sequence (I'm not sure if that can happen for the reader, but maybe) then the length will be `0` which is also a falsey value, so this condition will treat it the same as `None`.